### ISSUE TITLE
Unable to move while the mod is enabled

### ISSUE THREAD

**Issue by Katzenwerfer** [TeamDoodz/UKAmmoMod#3]
This is the error I get on the BepInEx log
```log
[Error  : Unity Log] Unable to open archive file: C:/Users/Katzenwerfer/AppData/Roaming/r2modmanPlus-local/ULTRAKILL/profiles/ULTRAKILL/BepInEx/plugins/TeamDoodz-UKAmmoMod/ukammomod_assets
[Error  : Unity Log] Failed to read data for the AssetBundle 'C:\Users\Katzenwerfer\AppData\Roaming\r2modmanPlus-local\ULTRAKILL\profiles\ULTRAKILL\BepInEx\plugins\TeamDoodz-UKAmmoMod\ukammomod_assets'.
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
UKAmmoMod.ModAssets.get_MainBundle () (at <4ff29cdf4642495f9fcedddc30b067f9>:0)
UKAmmoMod.Patches.<ShowAmmoUIPatch>F02779CDC7D38605D5CA58724F0E4D2859338F72460CDAD630D63832580F12E3B__ShowAmmoUIPatch.Prefix (PlayerActivator __instance) (at <4ff29cdf4642495f9fcedddc30b067f9>:0)
PlayerActivator.OnTriggerEnter (UnityEngine.Collider other) (at <ba0ba0851e804e1ab6533ea5a3d4d7bb>:0)

```

**Comment by Fuchsium** [comment]
Do you have UMM installed? UKAmmoMod requires UMM present to work but I think I forgot to add it as a dependency

**Comment by Katzenwerfer** [comment]
Yes, I have it installed, but it doesn't seem like r2modman understands how to install UMM mods, which results in them getting installed as a normal BepInEx plugin. Should add a note in your mods' description for that, I guess.

**Comment by Fuchsium** [comment]
I apologize, I mixed up UKAmmoMod with a different project. Ammo mod does not require UMM to function.

**Comment by Fuchsium** [comment]
After doing some testing it seems that ammo mod is actually incompatible with UMM, since they both try to access the game's common bundle. I'll fix this soon

**Comment by Fuchsium** [comment]
Fixed in 1.1.2
